ironmq/src/main/scala/akka/stream/alpakka/ironmq/CommittableMessage.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ironmq
+
+import akka.Done
+
+import scala.concurrent.Future
+
+/**
+ * Commit an offset that is included in a [[CommittableMessage]].
+ */
+trait Committable {
+  def commit(): Future[Done]
```

Comment:
I am expecting people will use the `scala.compat.java8.FutureConverters` to convert it to a `CompletionStage[Done]`.

```
import akka.Done
import java.util.concurrent._
import scala.compat.java8.FutureConverters._

val committable: Committable = new Committable {
  def commit(): Future[Done] = Future(Done)
}

val done: CompletionStage[Done] = committable.commit().asJava
```

I can add two methods `commitJava` and `commitScala` but I really don't like it. In that case, I will prefer having a different `Committable` for Java.